site/src/components/Carousel.js

```diff
@@ -135,27 +134,27 @@ export default function Carousel({itemList, title}) {
     }
 
     return (
-        <div className={carouselStyles.carouselContainer}>
-            <div className={`${carouselStyles.decoration} ${carouselStyles.top}`}>
+        <div className="carousel-container">
+            <div className="decoration top">
                 <Svg/>
             </div>
-            <div className={carouselStyles.content}>
-                <div className={carouselStyles.header} data-cy="header">{title}</div>
-                <div className={carouselStyles.carouselContent} style={carouselContentStyle}>
-                    <div className={carouselStyles.carousel}>
-                        <div className={`${carouselStyles.carouselSlides} ${hasTransitionClass ? carouselStyles.transition :""}`} style={carouselSlidesStyle} data-cy="carouselSlides">
+            <div className="content">
+                <div className="carousel-header" data-cy="carouselHeader">{title}</div>
+                <div className="carousel-content" style={carouselContentStyle}>
+                    <div className="carousel">
+                        <div className={`carousel-slides ${hasTransitionClass ? "transition" :""}`} style={carouselSlidesStyle} data-cy="carouselSlides">
                             {newItemList.map((props, idx) => (
                                 <CarouselItem key={idx} {...props} index={idx}/>
                             ))}
                         </div>
-                        <div className={`${carouselStyles.navButtons} ${navDisabled ? carouselStyles.disabled :""}`} style={navButtonsStyle}>
-                            <button className={carouselStyles.leftButton} onClick={() => scrollCarousel(currentIndex - 1)} style={buttonStyle} data-cy="leftButton"></button>
-                            <button className={carouselStyles.rightButton} onClick={() => scrollCarousel(currentIndex + 1)} style={buttonStyle} data-cy="rightButton"></button>
+                        <div className={`nav-buttons ${navDisabled ? "disabled" :""}`} style={navButtonsStyle}>
+                            <button className="left-button" onClick={() => scrollCarousel(currentIndex - 1)} style={buttonStyle} data-cy="leftButton"></button>
+                            <button className="right-button" onClick={() => scrollCarousel(currentIndex + 1)} style={buttonStyle} data-cy="rightButton"></button>
                         </div>
                     </div>
                 </div>
             </div>
-            <div className={`${carouselStyles.decoration} ${carouselStyles.bottom}`}>
+            <div className="decoration bottom">
```

Comment:
Is it good practice to have class names with spaces in them?

Reply:
Genuine question, not being snarky - ive just always seen them with `-` or `_`

Reply:
That decoration class was used for both the top and bottom decorations (amoeba). I changed the css to use an `@extend` and was able to remove the decoration class. I also renamed the `top` and `bottom` to `top-amoeba` and `bottom-amoeba` respectively.